platform/android/MapboxGLAndroidSDKTestApp/src/main/AndroidManifest.xml

```diff
@@ -991,6 +991,11 @@
          android:name="com.mapbox.MeasureTileDownloadOn"
          android:value="false" />
 
+        <!-- Set value to true to have tile loading measurements on -->
```

Comment:
SKU token generation doesn't have to do with tile loading measurement, does it? Update this comment? Seems leftover from the meta-data tag above it...

Reply:
Good catch - just did, definitely left over.